backends/sycl-gen/ceed-sycl-gen-operator-build.sycl.cpp

```diff
@@ -24,6 +24,8 @@
 
 #include "ceed-sycl-gen.hpp"
 
+#define SUB_GROUP_SIZE_OP 16
```

Comment:
Let's turn this into a variable with a default value of 16 so that it can be set programmatically in the future&mdash;e.g., changed if 16 isn't one of the subgroup sizes supported by the device.